docs/reference/aggregations/bucket/histogram-aggregation.asciidoc

```diff
@@ -317,3 +317,26 @@ Response:
     }
 }
 --------------------------------------------------
+
+==== Missing value
+
+The `missing` parameter defines how documents that miss a value should be treated.
+By default they will be ignored but it is also possible to treat them as if they
+had a value.
+
+[source,js]
+--------------------------------------------------
+{
+    "aggs" : {
+        "quantity" : {
+             "histogram" : {
+                 "field" : "quantity",
+                 "interval": 10,
+                 "missing": 0 <1>
+             }
+         }
+    }
+}
+--------------------------------------------------
+
+<1> Document that don't have a value will be in the same bucket as a document that would have `0` as a value
```

Comment:
Documents without a value in the `quantity` field will fall into the same bucket as documents that have the value `0`.